pkg/manager/modelregistry.go

```diff
@@ -48,12 +50,22 @@ func (m *Manager) RegisterModelPlugin(moduleName string) (string, string, error)
 	modelPlugin, ok := symbolMP.(ModelPlugin)
 	if !ok {
 		log.Warning("Unable to use ModelPlugin in ", moduleName)
-		return "", "", fmt.Errorf("Symbol loaded from module %s is not a ModelPlugin",
+		return "", "", fmt.Errorf("symbol loaded from module %s is not a ModelPlugin",
 			moduleName)
 	}
 	name, version, _, _ := modelPlugin.ModelData()
-	modelName := toModelName(name, version)
+	modelName := ToModelName(name, version)
 	m.ModelRegistry[modelName] = modelPlugin
+	modelschema, err := modelPlugin.Schema()
+	if err != nil {
+		log.Warning("Error loading model plugin", err)
```

Comment:
this is actually more reading model plugin schema. the model plugin has already being loaded.

Reply:
done